docs/UserGuide.md

```diff
@@ -3,7 +3,22 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Welcome to ResearchRoster! **Your Research, Simplified.**
+
+Designed with researchers in mind. ResearchRoster simplifies participant management so you can focus on what matters most: your research. Easily add and delete participants as required, conduct random sampling with a simple command, and export contacts for seamless communication.
```

Comment:
I thought "random sampling" is not the correct term for what we are doing in our assign command?